.github/workflows/pr.yml

```
@@ -96,7 +96,7 @@ jobs:
         fi
         
         dfpl convert -f tests/data
-        if [ "$(find tests/data \(-name '*.csv'\ -o -name '*.tsv' \) | wc -l)" -ne "$(find tests/data -name '*.pkl' | wc -l)" ]; then
+        if [ "$(find tests/data \(-name '*.csv' -o -name '*.tsv' \) | wc -l)" -ne "$(find tests/data -name '*.pkl' | wc -l)" ]; then
```

should be (but not sure if it helps with the problem

```suggestion
        if [ "$(find tests/data \(-name '*.csv' -o -name '*.tsv'\) | wc -l)" -ne "$(find tests/data -name '*.pkl' | wc -l)" ]; then
```

Also if you replace `set -e` by `set -ex` we will see all the commands that are executed which might help.

---

Now worked, there was a space missing.